docs/data.rst

```diff
@@ -1099,6 +1099,43 @@ Response
     {"status_code": 200, "message": "3 records were deleted"}
 
 
+Permanent Deletion of Submissions
+------------------------------------
+
+**Permanently Delete a specific submission instance**
+
+`DELETE /api/v1/data/{pk}/{dataid}`
```

Comment:
Seems like `curl -X DELETE https://api.ona.io/api/v1/data/28058' -d 'instance_ids=101425 -d permanent_delete='True'` will still work? Do we need this endpoint?

Reply:
Yes the above should work given instance_ids and permanent_delete params are set as payload params and not query params